ray_provider/xcom/ray_backend.py

```diff
@@ -31,11 +32,29 @@ def get_or_create_kv_store(identifier, allow_new=False):
 
 
 class KVStore:
+
     @ray.remote
-    class _KvStoreActor(object):
-        def __init__(self):
+    class _KVStoreActor(object):
```

Comment:
@richardliaw do you foresee any problems if we launch multiple of this actor at the same time or was that more around older versions of ray?

Reply:
Is it not intended to be a singleton and hence should be guarded against having multiple copies?

Reply:
@grechaw yes but I think that the file lock should fix that right?